Networking/NetworkingTests/Remote/CustomerRemoteTests.swift

```diff
@@ -0,0 +1,46 @@
+import XCTest
+@testable import Networking
+
+class CustomerRemoteTests: XCTestCase {
+
+    /// Dummy Network Wrapper
+    ///
+    private var network: MockNetwork!
+
+    /// Dummy Site ID
+    ///
+    private let sampleSiteID: Int64 = 123
+
+    /// Dummy Customer ID
+    private let sampleCustomerID: Int64 = 25
+
+    override func setUp() {
+        super.setUp()
+        network = MockNetwork()
+    }
+
+    override func tearDown() {
+        network = nil
+        super.tearDown()
+    }
+
+    /// Verifies that retrieveCustomer  properly parses the `wc/v3/customers/{customerID}` endpoint sample response.
+    ///
+    func test_retrieveCustomer_returns_parsed_customer_successfully() throws {
+        // Given
+        let remote = CustomerRemote(network: network)
+        network.simulateResponse(requestUrlSuffix: "customers/\(sampleCustomerID)", filename: "customer")
+
+        // When
+        let result = waitFor { promise in
+            remote.retrieveCustomer(for: self.sampleSiteID, with: self.sampleCustomerID) { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        XCTAssert(result.isSuccess)
+        let customer = try XCTUnwrap(result.get())
+        XCTAssertNotNil(customer)
```

Comment:
I think this assert is unnecessary: the previous one will fail and throw if it's nil. You may want to check another property about the customer instead, or nothing at all.

Reply:
I see what you mean, done at 32dfe2a623957706b5e8eb660430e8f2490c9d71

Reply:
Sorry, I meant the `XCTAssertNotNil` is unnecessary, though you're right that the `XCTAssert(result.isSuccess)` is also tested by the `XCTUnwrap`.

To be honest, and on second thoughts, since you never use the `customer` other than checking that it's there, unwrapping it isn't needed, I would consider simply doing `XCTAssertNotNil(result.get())` in place of all three, though perhaps with the unwrap it's clearer that we're looking at a customer.

This is super trivial, no need to change it, I'm only commenting again because I'd not been clear the first time.